foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchQueryStore.java

```diff
@@ -183,15 +184,19 @@ public Document get(String table, String id) throws FoxtrotException {
         }
         fxTable = tableMetadataManager.get(table);
         String lookupKey;
+        /**
+         * Need to see what function should be used to replace setNoFields
+         * setNoFields
+         */
         SearchResponse searchResponse = connection.getClient()
                 .prepareSearch(ElasticsearchUtils.getIndices(table))
                 .setTypes(ElasticsearchUtils.DOCUMENT_TYPE_NAME)
                 .setQuery(boolQuery().filter(termQuery(ElasticsearchUtils.DOCUMENT_META_ID_FIELD_NAME, id)))
-                .setNoFields()
+//                .setNoFields()
```

Comment:
Reference: https://stackoverflow.com/questions/42789560/elasticsearch-upgrade-from-1-to-5-searchrequestbuilder-setnofields

Reply:
Kar de ye.